docker/ConfigureDocker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Create the docker "image"; this is the tempate that containers are created from.
+# Both images are tagged with pipetag; this will help prevent multiple overlapping instances from running.
+# This file should create both images
+
+# CLI
+# Dockerfile is located in subdirectory dockerfiles\cli
+
+sudo docker build -t cli \
+	.\cli\
```

Comment:
I think that pretty much all of the paths in this script need to be changed. From what I can tell of the structure for the project now it seems that you need to move up a directory and change the directory names to all caps like so.
`../CLI \` or `../GUI \`. Also note that your back slashes need to change to forward slashes.